content/source/docs/plugin/framework/publishing.html.md

```diff
@@ -15,21 +15,35 @@ version 5 of the Terraform protocol, but providers built using the framework are
 
 ## Add a Version Manifest
 
-When you upload your new provider version to GitHub:
-
-1. Next to the zip files containing your binaries for each platform, include a file named `terraform-provider-$NAME_$VERSION_manifest.json` where `$NAME` is your provider's name (e.g. `random`) and `$VERSION` is your provider's version (e.g. `1.2.3`).
+Add the following contents to a file (e.g. `terraform-registry-manifest.json`). The version number in the file is the version of the manifest format, not the version of your provider.
```

Comment:
I want to push back here on this rearrange only because like this users might name the file the wrong thing which is annoying. I wonder if we can move the information about how to name the file back up to be with this content and then change the below to:

When you upload your new provider version to GitHub:
1. Add the manifest file next to the zip files containing your binaries for each platform.
2. Include the SHA-256 checksum of this JSON file in your `SHA256SUMS` file.